### PR TITLE
Bump to v1.26.2

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
As the title says.  Bumping to v1.26.2

[NO NEW TESTS NEEDED]

Signed-off-by: tomsweeneyredhat <tsweeney@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@flouthoc @rhatdan @nalind PTAL

File: contrib/rpm/buildah.spec [hunk 1]
```diff
@@ -100,6 +100,9 @@ make DESTDIR=%{buildroot} PREFIX=%{_prefix} install install.completions
 %{_datadir}/bash-completion/completions/*
 
 %changelog
+* Wed Jun 30, 2022 Tom Sweeney <tsweeney@redhat.com> 1.26.2-1
```
Comment (nalind) [inline contrib/rpm/buildah.spec:103]: rpmbuild will complain about that date being a Thursday.

**Review by flouthoc — APPROVED** [review]
LGTM apart from the nit above.

**Comment by TomSweeneyRedHat** [comment]
Just repushed, sorry about the slow turn around, I missed the update.

**Comment by flouthoc** [comment]
@TomSweeneyRedHat We would need a tag for this release so we can vendor this into podman. Could you please cut a tag, not sure if i can do that.

**Comment by flouthoc** [comment]
Thanks @nalind

**Comment by TomSweeneyRedHat** [comment]
Thanks @nalind, and apologies for the tardiness @flouthoc

**Comment by flouthoc** [comment]
@TomSweeneyRedHat no issues at all :)

**Pull request merged**